### DISCUSSION TITLE
Update tif file

### DISCUSSION THREAD

**Discussion by UsuarioFabio** [opengisch/QField#4853] [Q&A]
Hi. I have a questions for you. 
I have a license of qgis field cloud. When I create a project with raster files (.tif) I have no problem but if I put a new raster in the project is imposible to update from the cloud to the phone Because fail all raster file (from notebook to the cloud it does not show any warning). 
follow the warning when I try to update project in phone:
Proyecto e8c22.......: la capa empaquetada 'file' no es válida.
Código de error invalid_dataprovider, mensaje de error:
Cannot open GDAL dataset /route in my notebook/file.tif 
I am something wrong or is not permited raster update ?
Best.
Fabio

**Comment by SeqLaz** [comment]
Hello Fabio,

Regarding the raster format, we recommend to using GeoPackage (.gpkg). Exporting the raster to this formats in QGIS and saving it as a new file will help ensure smoother synchronization and reduce potential errors, to export right click the layer Export > Save layer as > GeoPackage make sure you are saving this file to the working directory of the project.

**Comment by UsuarioFabio** [comment]
Thank for answer my questions.
I have tried with the tif and gpkg extensions.
I save the new raster to the folder where I had the project data C:\Users\User\Documents\QField\Folder and also to the C:\Users\User\QField\cloud\Folder folder to test.
When I ask to synchronize the project, the new raster is uploaded to the cloud but it does not show the need for an update on the phone and it is not updated on the cell phone.
![update](https://github.com/opengisch/QField/assets/115658304/c7c29830-3e58-4a85-b5c3-f24f536207a4)
![image](https://github.com/opengisch/QField/assets/115658304/2cf47631-6130-4349-b13a-54688fb3dc77)
Please could you tell me what am I doing wrong?

Best
Fabio

**Reply by seades0630** [reply]
Hello Fabio,

I have been encountering a very similar issue when trying to remove and/or add .tif or .gpkg files to existing projects.  I was receiving the same "invalid_dataprovider" error code.  The following solution worked for me, and the issue is yet to resurface.

I realized that I was using Qgis build 3.34.2, and apparently the current qfield/qcloud versions are running on 3.32.1 or something like that.  I reassembled my project using the 3.28.14 long term release, then uploaded the project again to qcloud, and this seems to have resolved the "invalid_dataprovider" issue.

If you are using a newer build than 3.32, I would recommend trying to recreate your project with an earlier version of Qgis and then try again.  Hope this works for you!

**Reply by UsuarioFabio** [reply]
Thank you seades0630.

**Comment by SeqLaz** [comment]
Hello Fabio.

For better assistance could you reach out to us opening a ticket on the question mark in the right corner of the QFieldCloud platform?

Also, if you can attach a sample of the project this would be better for assisting you further.

Best regards.
